packages/compass-crud/test/aggrid-helper.ts

```diff
@@ -189,4 +189,4 @@ module.exports = {
   NUM_DOCS: NUM_DOCS,
   expectedDocs: expectedDocs,
   checkPageRange: checkPageRange,
-};
+} as any;
```

Comment:
Why do we want to export it as any? Does it brake something where it is being imported?

Reply:
It’s a test helper that contains mocks for use in tests, so this isn’t really expected to match the types in the places where it’s being used